docs/_posts/gadde5300/2022-12-15-finner_10q_xlbr_md_subset10_en.md

```diff
@@ -2,9 +2,9 @@
 layout: model
 title: Finance NER (10Q, md, 12 entities)
 author: John Snow Labs
-name: finner_10q_xlbr_md_subset10
+name: finner_10q_xblr_md_subset10
```

Comment:
Please pay attention. The correct one is [XBRL](https://en.wikipedia.org/wiki/XBRL).